arm-sql/2014-04-01/examples/DataMaskingRuleCreateOrUpdateDefaultMax.json

```diff
@@ -0,0 +1,69 @@
+{
+   "parameters": {
+      "subscriptionId": "00000000-1111-2222-3333-444444444444",
+      "resourceGroupName": "sqlcrudtest-6852",
+      "serverName": "sqlcrudtest-2080",
+      "databaseName": "sqlcrudtest-331",
+      "dataMaskingPolicyName": "Default",
+      "api-version": "2014-04-01",
+      "dataMaskingRuleName": "rule1",
+      "parameters": {
+         "properties": {
+            "aliasName": "nickname",
```

Comment:
If it's being returned by the API, needs to be documented as a deprecated parameter.